sources/api/migration/migration-helpers/src/common_migrations.rs

```diff
@@ -295,97 +295,110 @@ impl Migration for ReplaceStringMigration {
 
 // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
-/// We use this migration when we replace a setting that contains a list of string values.
+/// We use this migration when we replace settings that contains a list of string values.
 // String is the only type we use today, and handling multiple value types is more complicated than
 // we need at the moment.  Allowing &[serde_json::Value] seems nice, but it would allow arbitrary
 // data transformations that the API model would then fail to load.
-pub struct ReplaceListMigration {
+
+pub struct ListSetting {
```

Comment:
```suggestion
pub struct ListReplacement {
```
This seems a bit closer to its purpose, and I think `replacement.setting` reads clearer than `list_setting.setting` below, for example...